core/src/main/scala/weaponregex/parser/ParserJS.scala

```diff
@@ -20,7 +20,7 @@ import weaponregex.model.regextree.*
 class ParserJS private[parser] (pattern: String, val flags: Option[String] = None) extends Parser(pattern) {
 
   /** Whether the flags contain the `u` flag for Unicode mode */
-  private val unicodeMode: Boolean = flags.contains("u")
+  private val unicodeMode: Boolean = flags.exists(_.contains("u"))
```

Comment:
Does this mean that `/\u{20}/gu` isn't currently supported for JavaScript?

Reply:
Yes, the bug is it will only find flags if it's the only flag in the regex

Reply:
... but you're using `flags.contains` right? Pretty sure it should look for `"u"` anywhere in the string...?

Reply:
Another question: how can you provide more than 1 `flags` option? Since `flags` is of type `Option[String]` ?

Reply:
> ... but you're using flags.contains right? Pretty sure it should look for "u" anywhere in the string...?

It would if it's a string. But the flags is a `Option[String]`, so `contains` just checks if the value inside the `Option` is equal to the param. Instead we want to check if the `Option` is defined, and the string inside contains `"u"`. That's what the `.exist(_.contains(...))` does. 

> Another question: how can you provide more than 1 flags option? Since flags is of type Option[String] 

By having a longer String :) that contains the flag character in it

Reply:
Omg, that is very confusing and an easy mistake to make! Feel free to merge and release 😉